### PR TITLE
Adjusting Android styles from v9 to v10

### PR DESCRIPTION
**Pull request by langsmith**
Bumped them up to v10

cc @cammace

### CODE REVIEW

File: platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/constants/Style.java [hunk 1]
```diff
@@ -35,36 +35,36 @@
...
    */
-  public static final String OUTDOORS = "mapbox://styles/mapbox/outdoors-v9";
+  public static final String OUTDOORS = "mapbox://styles/mapbox/outdoors-v10";
 
   /**
    * Light: Subtle light backdrop for data visualizations. Using this constant means your map
    * style will always use the latest version and may change as we improve the style.
    */
-  public static final String LIGHT = "mapbox://styles/mapbox/light-v9";
+  public static final String LIGHT = "mapbox://styles/mapbox/light-v10";
```
Comment (1ec5) [inline platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/constants/Style.java]: There are no Dark, Light, or Satellite v10 styles.

**Comment by langsmith** [comment]
@1ec5, I think I should just delete this branch/pr because https://github.com/mapbox/mapbox-gl-native/pull/6301 seems to already address all of the Android changes needed for v9 –> v10 . Any objection?

**Comment by 1ec5** [comment]
No objection. Would you mind double-checking #6301 to make sure it takes care of everything that needs to change on the Android side?

**Comment by tobrun** [comment]
#6301 is looking great @1ec5, going ahead and closing this in favor of the all-platform update PR. Thanks @langsmith for willing to pick this up!

**Pull request closed without merging**